FeaturevisorDemo/FeaturevisorDemo/main.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  main.swift
+//  FeaturevisorDemo
+//
+//  Created by Marcin Polak on 11/08/2023.
+//
+
+import Foundation
+import FeaturevisorSDK
+import FeaturevisorTypes
+
+print("Hello, Featurevisor!")
+
+let instanceOptions = InstanceOptions(
+        datafileUrl: "https://your-datafile.json",
+        logger: createLogger())
+
+let featurevisor = createInstance(options: instanceOptions)
```

Comment:
if `FeaturevisorDemo` treated as a package or a separate app by having it as a different folder?

is it something worth recreating as a test case instead?

Reply:
I added it as a separate project to easily run and debug the SDK. Such demo projects are usually in a root folder of library/framework. However, now I think that for now, I won't push it. I will keep it locally and later we will decide. I will close this PR